README.md

```diff
@@ -20,153 +20,44 @@ analysis of the different steps and combinations of preprocesssing and integrati
 
 ### Please cite:
 
-_**Benchmarking atlas-level data integration in single-cell genomics.**  
-MD Luecken, M Büttner, K Chaichoompu, A Danese, M Interlandi, MF Mueller, DC Strobl, L Zappia, M Dugas, M Colomé-Tatché, FJ Theis
-bioRxiv 2020.05.22.111161; doi: https://doi.org/10.1101/2020.05.22.111161 _
+Luecken, M.D., Büttner, M., Chaichoompu, K. et al. Benchmarking atlas-level data integration in single-cell genomics.
+Nat Methods 19, 41–50 (2022). https://doi.org/10.1038/s41592-021-01336-8
 
 ## Installation
 
-To reproduce the results from this study, three different conda environments are needed.
-There are different environments for the python integration methods, the R integration methods and
-the conversion of R data types to anndata objects.
+To reproduce the results from this study, two separate conda environments are needed for python and R operations.
+Please make sure you have [`conda`](https://conda.io/projects/conda) installed on your system to be
+able to use the pipeline.
+We also recommend installing [`mamba`](https://mamba.readthedocs.io) for shorter installation times and smaller memory
+overhead.
 
-The main steps are:
+The installation is automated by an installation script that install the correct python and R environments based on the
+R version you want to use.
+The pipeline currently supports R 3.6 and R 4.0, and we generally recommend using the latest R version.
```

Comment:
Ah, so I was trying to say that people should prefer 4.0 over 3.6 in this case. Would you still recommend R 3.6 over R 4.0? From experience updated packages tend to break older R environments, so I guess people will usually have more luck installing to newer R versions. Also, newer projects might want to use newer package versions.

Reply:
Let's recommend 4.0 over 3.6... but not make a comment about newer versions for now. What you say might be true, it might also require us to update the code for newer versions of R

Reply:
OK, I see the ambiguity. I clarified it now.